packages/core/src/main/Breadcrumbs/storybook/collapse/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Story } from '@storybook/react/types-6-0';
+import { Breadcrumbs, BreadcrumbsProps } from '@core';
+import { Column, Row, Title } from '@core/storybook/components';
+import { setStoryParams } from '@core/storybook/setStoryParams';
+
+export const CollapseStory: Story<BreadcrumbsProps> = (props) => (
+    <Row>
+        <Column>
+            <Title>
+                Отображение по умолчанию
+            </Title>
+            <Breadcrumbs
+                {...props}
+                collapse={false}
+            />
+        </Column>
+        <Column>
+            <Title>
+                Предустановка по сокрытию. При соблюдении условия
+                <b> не менее 5 элементов. </b>
+            </Title>
+            <Breadcrumbs
+                {...props}
+                collapse
+            />
+        </Column>
+        <Column>
+            <Title>
+                Кастомизация сокрытия. В данном случае со 2 до 6 элементов.
+                Сокрытие определяется по
+                <b> индексу элемента </b>
+                в массиве.
+            </Title>
+            <Breadcrumbs
+                {...props}
+                collapse={{ start: 2, end: 6 }}
+            />
+        </Column>
+    </Row>
+);
+
+setStoryParams(CollapseStory, {
+    title: 'Сокрытие промежуточных элементов',
```

Comment:
```suggestion
    title: 'Скрытие промежуточных элементов',
```
"Сокрытие" как-то по судебному звучит)

Reply:
Там еще в нескольких местах по тексту есть "сокрытие", имхо лучше везде в "скрытие" переименовать